docker/Dockerfile.cpu

```diff
@@ -0,0 +1,67 @@
+FROM ubuntu:18.04
```

Comment:
I think it can be removed, since it is not used anywhere

Reply:
`Dockerfile.cpu` removed.